### DISCUSSION TITLE
Am I dumb ? no route no traffic

### DISCUSSION THREAD

**Discussion by stanthewizzard** [wg-easy/wg-easy#366] [Q&A] [answered]
Hello

My wg-easy sits on photon. iptable maybe restricitve (only ping is allowed)

On top of the network I have a routeur
route 10.7.0.0/24 (wg) through gateway 192.168.0.25 (fixed ip of photon)

I have no access to lan from wg. I need to give the route to wg to 192.168.0.25 ? from the container ?
thanks for help

**Comment by pheiduck** [comment, marked as answer]
Hey @stanthewizzard please try the latest stable image: `ghcr.io/wg-easy/wg-easy:latest` and report back if this is still a thing. :)

**Reply by stanthewizzard** [reply]
sadly still a thing
I'm using https://github.com/linuxserver/docker-wireguard
working flawlessly

I'm behind a double nat
maybe this is why ?
thanks

**Reply by stanthewizzard** [reply]
EDIT: wortking. I change in the compose 51820 to other port
    ports:
      - "51920:51820/udp"
the wg0.conf is still in 51820
After scaning the QR code I need to change the settings to 51920

**Reply by eandersons** [reply]
You also need to set the environment variable `WG_PORT` so the custom port would be used in the new clients' configuration.

**Reply by stanthewizzard** [reply]
i changed the WG_PORT in compose with no effect on the wg0.conf
and when changed manually after container restart the wg0 is reseted to 51820

**Reply by eandersons** [reply]
`WG_PORT` does not affect `wg0.conf`, it is only used in clients' configuration that can be downloaded or scanned from QR code.

**Reply by stanthewizzard** [reply]
ok thanks
